autoload/neomake.vim

```diff
@@ -42,6 +42,13 @@ function! neomake#GetJob(job_id) abort
     return s:jobs[a:job_id]
 endfunction
 
+function! neomake#GetErrors(bufnr) abort
+    if !a:bufnr
+        return s:current_errors['project']
+    endif
+    return get(s:current_errors['file'], a:bufnr, {})
+endfunction
+
```

Comment:
I'm not sure what you mean.  Are you saying that you want it to be more flexible and compatible with the goal of that airline issue?